app/assets/stylesheets/landing.scss

```diff
@@ -5,18 +5,56 @@ $black: #000000;
 $light-gray: #e0e0e0;
 $pale-orange: #fff4e1;
 $search-box: #f5f5f5;
+$white: #fff;
```

Comment:
@andela-tfowotade can we make this #ffffff to be consistent? we should probably decide if we want our color codes to be written in upper or lowercase